ext/standard/tests/math/pow_variation1.phpt

```diff
@@ -62,18 +62,20 @@ $inputs = array(
 /*20*/ "abcxyz",
        'abcxyz',
        $heredoc,
+       '5.5',
+       '2',
 
        // object data
-/*23*/ new classA(),
+/*25*/ new classA(),
 
        // undefined data
-/*24*/ @$undefined_var,
+/*26*/ @$undefined_var,
 
        // unset data
-/*25*/ @$unset_var,
+/*27*/ @$unset_var,
 
        // resource variable
-/*26*/ $fp
+/*28*/ $fp
 );
 
 // loop through each element of $inputs to check the behaviour of pow()
```

Comment:
Can't the two variation tets just be combined into 1?

Running a double foreach loop seems more sensible? Also, not sure having the iteration numbers is that useful. Compared to knowing what were the operands.

A bunch of the "data" is also repeated, we don't care about the capitalization of ``null`` or the boolean values

Reply:
I have removed all redundant use cases. I left two separate tests but may combine it later. I just wanted to add missing numerical strings to the tests.